js/RecentDonations.js

```diff
@@ -0,0 +1,63 @@
+
+var app = angular.module('RecentDonations', []);
+app.controller('RecentDonationsControl', function($scope) {
+    
+
+    GLOBALS.GET_PARTICIPANT_INFO(160204, function(data) {
+    	$scope.displayName = data["firstName"]
+    })
+
+    $scope.contributors = [{
+    				name:"...", 
+    				amount:"...",
+    				message:"...",
+    				date:"..."}]
+
+    $scope.update = function() {
+    	GLOBALS.GET_PARTICIPANT_DONATION_INFO(160204, function(data) {
+
+    		// Success, hide any error message
+    		$scope.error = undefined
```

Comment:
you say it's a string.... so maybe define it as ```''```? seeing something defined to undefined just weirds me out. but it's also okay probably?